src/xdocs/config_whitespace.xml

```diff
@@ -2437,49 +2437,49 @@ try {
           </tr>
           <tr>
             <td>allowEmptyConstructors</td>
-            <td>allow empty constructor bodies</td>
+            <td>Allow empty constructor bodies.</td>
             <td><a href="property_types.html#boolean">Boolean</a></td>
             <td><code>false</code></td>
             <td>4.0</td>
           </tr>
           <tr>
             <td>allowEmptyMethods</td>
-            <td>allow empty method bodies</td>
+            <td>Allow empty method bodies.</td>
             <td><a href="property_types.html#boolean">Boolean</a></td>
             <td><code>false</code></td>
             <td>4.0</td>
           </tr>
           <tr>
             <td>allowEmptyTypes</td>
-            <td>allow empty class, interface and enum bodies</td>
+            <td>Allow empty class, interface and enum bodies.</td>
             <td><a href="property_types.html#boolean">Boolean</a></td>
             <td><code>false</code></td>
             <td>5.8</td>
           </tr>
           <tr>
             <td>allowEmptyLoops</td>
-            <td>allow empty loop bodies</td>
+            <td>Allow empty loop bodies.</td>
             <td><a href="property_types.html#boolean">Boolean</a></td>
             <td><code>false</code></td>
             <td>5.8</td>
           </tr>
           <tr>
             <td>allowEmptyLambdas</td>
-            <td>allow empty lambda bodies</td>
+            <td>Allow empty lambda bodies.</td>
             <td><a href="property_types.html#boolean">Boolean</a></td>
             <td><code>false</code></td>
             <td>6.14</td>
           </tr>
           <tr>
             <td>allowEmptyCatches</td>
-            <td>allow empty catch bodies</td>
+            <td>Allow empty catch bodies.</td>
             <td><a href="property_types.html#boolean">Boolean</a></td>
             <td><code>false</code></td>
             <td>7.6</td>
           </tr>
           <tr>
             <td>ignoreEnhancedForColon</td>
-            <td>ignore whitespace around colon in for-each loops</td>
+            <td>Ignore whitespace around colon in enhanced for loops.</td>
```

Comment:
@rnveach , 

we need your native look here.
In documentation https://docs.oracle.com/javase/specs/jls/se11/html/jls-14.html#jls-14.14.2 "enhanced for" is used but `for` is in special formatting so it does not mean english word "for". 

I would rather use "enhanced-for loops" or `enhanced <i>for</i> loops` or `<a href="https://docs.oracle.com/javase/specs/jls/se11/html/jls-14.html#jls-14.14.2">enhanced for</a> loop` to be as as close as possible to Jdk specification ( refactoring from "loop" to "statement" is too much ).

Reply:
The JLS does `<code>for</code>`. I am fine with doing that in addition to using the `a href`.

Reply:
done.